docs/data/joy/components/skeleton/AnimationSkeleton.tsx

```diff
@@ -31,7 +31,7 @@ export default function AnimationSkeleton() {
             />
           </Skeleton>
         </AspectRatio>
-        <Typography>
+        <Typography sx={{ overflow: 'hidden' }}>
```

Comment:
Fixes #38211

Reply:
Is there a way to fix this in the Skeleton? I'm afraid developers will ship this bug in production without noticing.